### PR TITLE
End to end tests eram

### PR DESCRIPTION
**Pull request by EramM805**
Completed some end to end tests

### CODE REVIEW

File: e2e/logInSignUp.spec.js [hunk 1]
```diff
@@ -134,8 +135,22 @@ describe('Log in flow', () => {
 
 describe('testing swiping on feed page', () => {
```
Comment (bujars) [inline e2e/logInSignUp.spec.js:141]: Again (For some reason comments won't consistently appear, but we need better naming. Here we are not swiping through the feed but rather through the pages starting with feed!

File: src/screens/Feed.js [hunk 2]
```diff
@@ -108,6 +108,7 @@ export default class FeedScreen extends React.Component {
   //sample post data for ui testing purposes. We also have use react-native carousel for swiping posts.
   render() {
     console.log('Posts are:', this.state.posts);
+    console.disableYellowBox = true;
```
Comment (bujars) [inline src/screens/Feed.js]: Remove for dev purposes

Comment (bujars) [reply src/screens/Feed.js]: Possibly we need two different environments (or 3) dev/production/testing

File: src/screens/Search.js [hunk 3]
```diff
@@ -108,6 +108,7 @@ export default function SearchScreen() {
       />
       {showResults()}
       <Button
+      testID="logout-button"
```
Comment (bujars) [inline src/screens/Search.js]: Consistent naming

ie logout-button vs SVGProps

File: __tests__/LikeButton-test.js [hunk 4]
```diff
@@ -34,8 +34,8 @@ describe('testing handeLikeClick cases', () => {
 function testPropTypesWithoutErrors(isLiked) {
   const heart = 'heart';
   const heartOutline = 'heart-outline';
-  const colorRed = 'red';
-  const colorGrey = 'grey';
+  const colorCF6679 = '#cf6679';
```
Comment (sha256rma) [inline __tests__/LikeButton-test.js:37]: maybe have a e descriptive name

**Review by sha256rma — COMMENTED** [review]
nice tests. they go through the edge cases too!

File: src/svg/HomeSVG.js [hunk 5]
```diff
@@ -10,6 +11,7 @@ Animated.addWhitelistedNativeProps({
 
 const HomeSVG = ({color, size}: SVGProps) => {
   return (
+    <View testID="FeedSVG">
```
Comment (marvestt) [inline src/svg/HomeSVG.js:14]: Try to choose between using either camelCase or lisp-case for testIDs in order to maintain consistency

File: src/svg/LikeSVG.js [hunk 6]
```diff
@@ -10,6 +11,7 @@ Animated.addWhitelistedNativeProps({
 
 const LikeSVG = ({color, size}: SVGProps) => {
   return (
+    <View testID="UploadSVG">
```
Comment (marvestt) [inline src/svg/LikeSVG.js:14]: Choose between camelCase and lisp-case

File: src/svg/ProfileSVG.js [hunk 7]
```diff
@@ -11,6 +11,7 @@ Animated.addWhitelistedNativeProps({
 
 const ProfileSVG = ({color, size}: SVGProps) => {
   return (
+    <View testID="ProfileSVG">
```
Comment (marvestt) [inline src/svg/ProfileSVG.js:14]: camelCase or lisp-case
